RFCNNNN-New-RFC-Template.md

```diff
@@ -6,7 +6,18 @@ SupercededBy: <link to another RFC>
 Version: <Major>.<Minor>
 Area: <Area within the PowerShell language>
 Comments Due: <Date for submitting comments to current draft (minimum 1 month)>
-Plan to implement: <Yes | No>
+Plan to implement: <Yes | Yes but with help | No>
+# Helpers is only required if 'Plan to implement' is 'Yes but with help' and can contain either direct links to an individual or to a well known team like the Cmdlets working group or a project like PSResourceGet. This allows for both the "I wanna do this alone" as well as "I wanna do this with others" ways of working, whilst enabling opportunities to be able to work closer with others either as part of the PowerShell Team or from the Community. 
+Helpers: 
+# Helpers can be either individual accounts, a well known team like the working groups, an organisation or all of a project's repository maintainers
+ - kilasuit
```

Comment:
I'd rather not provide these examples otherwise people would default to them and it'll become unmanageable over time.  Instead, I get the idea that the author wants to implement but needs help.  I think it would be better to indicate the type of help needed and community or WGs can identify people or WGs that can assist if they agree on the value of the proposal.